src/transformer/gradient-css.ts

```diff
@@ -1,21 +1,27 @@
-import { Transform, TransformedToken } from 'style-dictionary/types'
-import { isGradient } from '../filter/isGradient.js'
+import { Transform, TransformedToken } from "style-dictionary/types";
+import { isGradient } from "../filter/isGradient.js";
+import { getValue } from "../utilities/getValue.js";
 
 type TokenGradient = {
-  color: number
-  position: number
-}
+  color: number;
+  position: number;
+};
 
 export const gradientCss: Transform = {
-  name: 'gradient/css',
+  name: "gradient/css",
   type: `value`,
   transitive: true,
   filter: isGradient,
   transform: (token: TransformedToken) => {
     // combine stops to string
-    const stops = token.value.map((stop: TokenGradient) => `${stop.color}${stop.position ? ` ${Math.floor(stop.position * 100)}%` : ""}`).join(", ")
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const stops = (getValue(token) as any[])
```

Comment:
Hey @AndreBarr Why do we need `any`? Couldn't we use `TokenGradient` instead to avoid disabling typescript?

Reply:
Updated